core/server/proxy/src/main/java/alluxio/proxy/s3/CompleteMultipartUploadHandler.java

```diff
@@ -87,6 +87,9 @@ public void handle(String s, Request request, HttpServletRequest httpServletRequ
       if (!s.startsWith(mS3Prefix)) {
         return;
       }
+      if (!request.getMethod().equals("POST") || request.getParameter("uploadId") == null) {
+        return;
+      } // Otherwise, handle CompleteMultipartUpload
```

Comment:
Please undo this change, at the moment we're relying on those logs for the _other_ S3 API actions too. If you want to, it'd be great to separate out these logs to a separate handler entirely.

Reply:
I do find that other S3 API will execute this code part. I'm just curious why it was designed like this. especially it's  `MultipartUploadHandler.handle()`. we may need a more common part (maybe a handler or filter?) to do this logging thing